ports/wasmedge/vcpkg.json

```
@@ -1,7 +1,6 @@
 {
   "name": "wasmedge",
-  "version-semver": "0.12.0-alpha.1",
-  "port-version": 1,
+  "version-semver": "0.12.1",
```

```suggestion
  "version": "0.12.1",
```
We don't have to use the syntax of `semver`, it only applies to `alpha`. Unless there is an explicit instruction to use `semver`.